src/source/source.js

```diff
@@ -50,7 +50,7 @@ exports.setType = function (name, type) {
  * @param {string} options.type The source type, matching the value of `name` used in {@link Style#addSourceType}.
  * @param {Dispatcher} dispatcher A {@link Dispatcher} instance, which can be used to send messages to the workers.
  *
- * @fires load to indicate source data has been loaded, so that it's okay to call `loadTile`
```

Comment:
Small addition: "@fires data **with `sourceDataType: 'metadata'`** to indicate..." -- since that's what source cache now requires of the event fired by `Source`.